inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/enums/PostgresFamily.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *   Licensed to the Apache Software Foundation (ASF) under one
+ *   or more contributor license agreements.  See the NOTICE file
+ *   distributed with this work for additional information
+ *   regarding copyright ownership.  The ASF licenses this file
+ *   to you under the Apache License, Version 2.0 (the
+ *   "License"); you may not use this file except in compliance
+ *   with the License.  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.apache.inlong.sort.protocol.enums;
+
+/**
+ *  database of based postgre
+ */
+public enum PostgresFamily {
+    POSTGRES,
+    TBASE,
```

Comment:
what is TBASE? please add more a description and usage link.

Reply:
I remove this enum. Add TDSQL Postgres load node and describe what is TDSQL Postgres.